modules/network-observability-flowcollector-about.adoc

```diff
@@ -0,0 +1,116 @@
+        src_namespace: 'openshift-,netobserv'
+        dst_namespace: 'openshift-,netobserv'
+    - name: Pods network
+      filter:
+        src_kind: 'Pod'
+        dst_kind: 'Pod'
+      default: true
+    - name: Services network
+      filter:
+        dst_kind: 'Service'
+----
+<1> The Agent specification, `spec.agent.type`, can be eBPF (default) or IPFIX. eBPF is the only {product-title} supported option, as it should work in more situations and offers better performance; IPFIX is a functional option but it is unsupported.
+<2> You can set the Sampling specification, `spec.agent.ebpf.sampling` and `spec.agent.ipfix.sampling`, to manage resources. Continuously sampled flow collection might consume a large amount of resources. You can mitigate this by setting a sampling ratio. A value of 100 means one flow every 100 is sampled. A value of 1 means all flows are sampled. The lower the value, the increase in returned flows and the accuracy of derived metrics. By default, eBPF sampling is set to a value of 50, for example 1:50. Note that more sampled flows also means more storage needed. It is recommend to start with default values and refine empirically, to determine which setting your cluster can manage.
+<3> The Loki specification, `spec.loki` configures how to reach Loki. The default values match the Loki install paths mentioned in the Installing the Loki Operator section, but you may have to configure differently if you used another installation method.
+
+= Flow Collector Configuration Parameters
+The fields for the Flow Collector custom resource are described in the following table.
+
+.FlowCollector custom resource configuration parameters
+[cols="1,1,8a",options="header"]
+|===
+
+|Field
+|Type
+|Description
+
+|`spec.agent.type`
+|`string`
+|Specifies the agent for flows extraction. eBPF is the default agent and is the only {product-title} supported option; IPFIX is a functional option but it is unsupported.
+
+|`spec.agent.ebpf`
+|`integer`
+|Specifies the the settings related to the eBPF-based flow reporter. The following eBPF settings are helpful for managing compute resources when continuously sampled flow collection might consume a large amount of resources:
+
+* `ebpf.sampling` specifies the sampling rate of the flow reporter. A value of 100 means one flow every 100 is sampled. A value of 1 means all flows are sampled. The lower the value, the increase in returned flows and the accuracy of derived metrics. The default value is 50. 
+* `ebpf.resources`specifies the compute resources required by the container. The default value is 800 MB, which is sufficient for most medium-sized clusters
+* `ebpf.cacheMaxFlows` and `ebpf.cacheActiveTimeout` specify how often flows are reported by the agents. The higher the `cacheMaxFlows` and `cacheActiveTimeout` values, the less traffic the eBPF agent generates, which reduces resource demands; however note that memory consumption is higher which may create latency in the flow collection.
```

Comment:
To advance some questions about the differences between both, I'd explain what they do:

* `cacheActiveTimeout` is a [Duration string](https://pkg.go.dev/time#ParseDuration) that specifies the maximum duration that flows are aggregated in the eBPF agent before being forwarded to the processor. Every `cacheActiveTimeout`, the Agent flushes the accounted flows to the Flow Processor.
* `cacheMaxFlows` is the maximum number of flows that can be kept in the eBPF agent memory. If the number of flows being aggregated reach this number, they are flushed to the Flow Processor even if the `cacheActiveTimeout` period is not reached.

Reply:
Thanks for these details!!